lessons/AI/ai-linear-regression/langs/en-us/index.adoc

```diff
@@ -0,0 +1,223 @@
+= Artificial Intelligence &  Regression
```

Comment:
This Title feels like it's setting people up to be disappointed. I suggest retitling the lesson Supervised Machine Learning. While regression is acknowledged as an available tool (and may be something the Bootstrap:AI pathway will teach in the future), this lesson does not teach regression.